packages/flutter_tools/test/integration.shard/android_plugin_example_app_build_test.dart

```diff
@@ -107,6 +118,15 @@ android.enableR8=true''');
       'apk',
       '--target-platform=android-arm',
     ], workingDirectory: exampleAppDir.path);
+    expect(appBuildDir.existsSync(), isFalse);
     expect(exampleApk, exists);
+  }
+
+  test('plugin example can be built using current Flutter Gradle plugin', () async {
+    _createAndBuildPlugin('plugin_test');
+  });
+
+  test('plugin example android/app/build should not exists after flutter build', () async {
```

Comment:
@blasten I have added a new test case here, should I move it to a standalone test file?